src/components/htmllayout.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
Is this a file for testing? If so, it should go. Layout is being done by @banciur.